src/test/data/JsonUserPrefsStorageTest/ExtraValuesUserPref.json

```diff
@@ -7,7 +7,8 @@
       "x" : 300,
       "y" : 100,
       "z" : 99
-    }
+    },
+    "isMaximized" : false
   },
```

Comment:
Should this be updated with the divider position?

Reply:
Yes, it has been updated